crates/oxc_parser/src/lexer/kind.rs

```diff
@@ -390,109 +390,10 @@ impl Kind {
     #[must_use]
     pub fn match_keyword(s: &str) -> Self {
         let len = s.len();
-        if len == 1 || len >= 12 {
+        if len <= 1 || len >= 12 {
```

Comment:
I'd expect this to result single branch, but if we want to ensure we can [use a trick](https://stackoverflow.com/a/17095534) just in case.

Can make it a small helper method `InRange` or w/e.
IIRC v8 does this.

Reply:
This is interesting one.